Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Beckn/ACL/OnUpdate.hs

```diff
@@ -135,6 +140,12 @@ buildOnUpdateMessage req@RideCompletedBuildReq {} = do
             computed_value = fare
           }
       breakup = mkBreakupList (OnUpdate.BreakupPrice currency . fromIntegral) OnUpdate.BreakupItem req.fareParams
+      buildStartLocation = case req.startLocation of
```

Comment:
Can be refactored:
`startLocation = mkLocation <$> req.startLocation`
We usually use `mk` or `make` prefix for pure functions, build prefix for monadic functions